client/src/routes/join/+page.svelte

```diff
@@ -24,6 +24,8 @@
   let codeDirty: boolean = false;
   $: codeValidation = lobbyCodeValidator(code);
 
+  let once: boolean = false;
```

Comment:
I think we could find a more descriptive variable name maybe `buttonActive` or `waiting` or something like that